packages/kbn-securitysolution-exceptions-common/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-common",
   "id": "@kbn/securitysolution-exceptions-common",
-  "owner": "@elastic/security-detection-engine"
-}
+  "owner": [
+    "@elastic/security-detection-engine"
+  ],
+  "group": "security",
```

Comment:
This is used within the lists plugin. Is the lists plugin part of the "security" group?